mvpa2/algorithms/group_clusterthr.py

```diff
@@ -457,9 +461,13 @@ def get_thresholding_map(data, p=0.001):
     p_index = int(len(data) * p)
     if p_index < 1:
         raise ValueError("requested probability is too low for the given number of samples")
-    # threshold indices are all in one row of the argsorted inputs
-    thridx = np.argsort(data, axis=0, kind='quicksort')[-p_index]
-    return data[thridx, np.arange(data.shape[1])]
+    if externals.versions['numpy'] >= '1.9.0':
+        return np.percentile(data, q=(1-p)*100, interpolation='higher', axis=0)
```

Comment:
Is this really faster than the other solution in the 'else' block? By how much? If the difference is small, then adding more logic makes this harder to maintain.